docs/PowerAuth-Push-Server-1.3.0.md

```diff
@@ -0,0 +1,19 @@
+# Migration from 1.2.0 to 1.3.0
+
+## Database Changes
+
+### Fixing Incorrect Table Type
+
+In Oracle and Postgres databases, the `push_campaign_user.user_id` column used incorrect numeric type `INTEGER`/`NUMBER(19)` instead of `VARCHAR(255)`. To fix the issue, run the following scripts.
```

Comment:
`uses`? If we are going to fix something it should still exists.

Reply:
👍 Fixed